csaf_2.0/guidance/sbom-matching-interface.md

```diff
@@ -0,0 +1,91 @@
+## SBOM Matching Interface
+
+This documents serves as a guidance how to implement an interface for a [CSAF SBOM matching matching system](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#9117-conformance-clause-17-csaf-sbom-matching-system).
+
+The matching interface defines several data structures. Its intention is to match the information on a CSAF data structure against a defined SBOM format. Different SBOM formats exist and can be mapped to this specification. The following three data structures need to be defined: 
+- `SBOMDatabase`: A database of SBOMs
```

Comment:
Nit: Please separate structurally different elements with an empty line, so that the eyes are sufficient to render the structure for the mind.